calico-policies/private-network-isolation/calico-v3/us-south/allow-egress-pods-private.yaml

```diff
@@ -66,65 +66,60 @@ spec:
   - action: Allow
     destination:
       nets:
-      # Nets for US South: hou02, mex01, sao01, sjc03, sjc04, dal10, dal12, dal13
```

Comment:
This comment line was helpful when services teams using kube-sample as reference point raised pr's for their clusters.

Reply:
I added the comments back in manually.  We'll have to add that into the tool that creates these as well